src/plugins/prebuilt-library/ai-wizard/kadence-ai-wizard.js

```diff
@@ -134,9 +134,12 @@ export function KadenceAiWizard( props ) {
 
 		// Submit wizard data on finish buttton click.
 		if (event.type === 'click' && event.target.classList.contains('components-wizard__primary-button')) {
-			handleSave();
 
-			if ( ! photographyOnly ) {
+			if (photographyOnly) {
+				handleEvent('collection_updated');
+				onWizardClose();
+			} else {
+				handleSave();
```

Comment:
By moving the handle save here doesn't that make it so any photo change won't be saved?

Reply:
Ah, you're right. I was trying to isolate the event. I've moved it outside that if block so it's handled in both cases (Photo collection updates _and_ Wizard data updates).